roles/node_install/tasks/main.yaml

```diff
@@ -152,6 +152,13 @@
     mode: "0664"
   become: True
 
+- name: Update the containerd config file
+  template:
+    src: "{{ role_path }}/templates/config.toml.j2"
```

Comment:
Can you rename this template file to `etc_containerd_config.toml.j2 or `containerd_config.toml.j2`.  As is the file in the template in the folder doesn't make it's purpose clear by the filename.  `config.toml.j2` could be for anything.